libparsers/tests/xml/dynamic_lighting.cpp

```diff
@@ -0,0 +1,93 @@
+#include "parsers/xml/dynamic_lighting.hpp"
+#include <doctest.h>
+#include <iostream>
+
+using namespace parsers::xml;
+using dynamic_lighting::lighting_info;
+using parsers::errors::error_t;
+using parsers::errors::errors_t;
+
+TEST_SUITE_BEGIN("libparsers - xml - dynamic_lighting");
+
+TEST_CASE("libparsers - xml - dynamic_lighting - test") {
+	std::string const test_value =
+	    "<?xml version=\"1.0\" encoding=\"utf - 8\"?>"
+	    "<openBVE xsi=\"http://www.w3.org/2001/XMLSchema-instance\" xsd="
```

Comment:
Put the namespace stuff back. Also you need a test without the surrounding OpenBVE nodes.